python/tests/test_tree_stats.py

```diff
@@ -40,8 +42,30 @@
 import tskit
 import tskit.exceptions as exceptions
 
+
 np.random.seed(5)
 
+# Notes for refactoring:
```

Comment:
These are notes about what happens in this file, in particular for a near-future me in refactoring the tests here.